IdentityCore/src/cache/MSIDToken.m

```diff
@@ -368,8 +368,8 @@ - (void)fillAdditionalServerInfoFromResponse:(MSIDTokenResponse *)tokenResponse
     if ([tokenResponse isKindOfClass:[MSIDAADTokenResponse class]])
     {
         MSIDAADTokenResponse *aadTokenResponse = (MSIDAADTokenResponse *)tokenResponse;
-        [serverInfo setObject:aadTokenResponse.extendedExpiresIn forKey:@"ext_expires_on"];
-        [serverInfo setObject:aadTokenResponse.speInfo forKey:MSID_TELEMETRY_KEY_SPE_INFO];
```

Comment:
same here

Reply:
Yes, both ext_expires_in and spe_info might be nil, that's why I changed it to setValue